entity-framework/core/what-is-new/ef-core-5.0/whatsnew.md

```diff
@@ -14,6 +14,202 @@ This page does not duplicate the [plan for EF Core 5.0](xref:core/what-is-new/ef
+    CONSTRAINT [PK_Dogs] PRIMARY KEY ([Id])
+);
+
+CREATE TABLE [Pets] (
+    [Id] int NOT NULL,
+    [Name] nvarchar(max) NULL,
+    CONSTRAINT [PK_Pets] PRIMARY KEY ([Id])
+);
+```
+
+Entity types can be mapped to different tables using mapping attributes:
+
+```c#
+[Table("Animals")]
+public class Animal
+{
+    public int Id { get; set; }
+    public string Species { get; set; }
+}
+
+[Table("Pets")]
+public class Pet : Animal
+{
+    public string Name { get; set; }
+}
+
+[Table("Cats")]
+public class Cat : Pet
+{
+    public string EdcuationLevel { get; set; }
+}
+
+[Table("Dogs")]
+public class Dog : Pet
+{
+    public string FavoriteToy { get; set; }
+}
+```
+
+Or using `ModelBuilder` configuration:
+
+```c#
+protected override void OnModelCreating(ModelBuilder modelBuilder)
+{
+    modelBuilder.Entity<Animal>().ToTable("Animals");
+    modelBuilder.Entity<Pet>().ToTable("Pets");
+    modelBuilder.Entity<Cat>().ToTable("Cats");
+    modelBuilder.Entity<Dog>().ToTable("Dogs");
+}
+```
+
+Documentation is tracked by issue [#1979](https://github.com/dotnet/EntityFramework.Docs/issues/1979).
+
+### Migrations: Rebuild SQLite tables
+
+Compared to other database, SQLite is relatively limited in its schema manipulation capabilities. For example, dropping a column from an existing table requires that the entire table be dropped and re-created. EF Core 5.0 Migrations now supports automatic rebuilding the table for schema changes that require it.
+
+For example, imagine we have a `Unicorns` table created for a `Unicorn` entity type:
+
+```c#
+public class Unicorn
+{
+    public int Id { get; set; }
+    public string Name { get; set; }
+    public int Age { get; set; }
+}
+```
+
+```sql
+CREATE TABLE "Unicorns" (
+    "Id" INTEGER NOT NULL CONSTRAINT "PK_Unicorns" PRIMARY KEY AUTOINCREMENT,
+    "Name" TEXT NULL,
+    "Age" INTEGER NOT NULL
+);
+```
+
+We then learn that storing the age of a unicorn is considered very rude, so let's remove that property, add a new migration, and update the database. This update will fail when using EF Core 3.1 because the column cannot be dropped. In EF Core 5.0, Migrations will instead rebuild the table:
+
+```sql
+CREATE TABLE "ef_temp_Unicorns" (
+    "Id" INTEGER NOT NULL CONSTRAINT "PK_Unicorns" PRIMARY KEY AUTOINCREMENT,
+    "Name" TEXT NULL
+);
+
+INSERT INTO "ef_temp_Unicorns" ("Id", "Name")
+SELECT "Id", "Name"
+FROM Unicorns;
+
+PRAGMA foreign_keys = 0;
+
+DROP TABLE "Unicorns";
+
+ALTER TABLE "ef_temp_Unicorns" RENAME TO "Unicorns";
+
+PRAGMA foreign_keys = 1;
+```
+
+Notice that:
+* A temporary table is created with the desired schema for the new table
+* Data is copied from the current table into the temporary table
+* Foreign key enforcement is switched off
+* The current table is dropped
+* The temporary table is renamed to be the new table
+
+Documentation is tracked by issue [#2583](https://github.com/dotnet/EntityFramework.Docs/issues/2583).
+
+### Flexible type/function mapping
```

Comment:
@AndriySvyryd @smitpatel Could one or both of you write either one combined or multiple different sections for the issues listed below? I'm not sure quite how to approach explaining this to people.

Reply:
https://github.com/dotnet/efcore/issues/21415 Can be ignored, this is very low-level